chainer_compiler/chainer_compiler.py

```diff
@@ -203,6 +203,34 @@ def _run_translator(translator, mc, inputs):
     return graph
 
 
+def _resolve_name_correspondence(mc, params, name, param_values):
+    for running_mean_name in ('avg_mean', 'running_mean'):
+        if not name.endswith(running_mean_name):
+            continue
+        parent = name[:-len(running_mean_name)]
+        mean = params[parent + 'beta']
+        for link in mc.links():
+            if not hasattr(link, 'avg_mean'):
+                continue
+            if any(id(p) == id(mean) for p in link.params()):
+                param_values.append(link.avg_mean)
+                return True
```

Comment:
I think it's better to return `link.avg_mean` instead of True / False and not to modify `param_values` in this function, so that caller will look more consistent.